framework/console/controllers/AssetController.php

```diff
@@ -326,6 +327,11 @@ protected function loadTargets($targets, $bundles)
     protected function buildTarget($target, $type, $bundles)
     {
         $tempFile = $target->basePath . '/' . strtr($target->$type, ['{hash}' => 'temp']);
+        if (!is_file($tempFile)) {
+            FileHelper::createDirectory(dirname($tempFile), 0755, true);
+            touch($tempFile);
```

Comment:
See the error message here https://github.com/yiisoft/yii2/issues/7473#issue-59109592
The file simply isn't there when `file_md5` wants to access it.